cmd/gqltool/main.go

```diff
@@ -7,27 +7,25 @@ import (
 	"log" // revive:disable-line
 	"os"
 
-	"github.com/stateful/runme/v3/internal/auth"
 	"github.com/stateful/runme/v3/internal/client"
 	"github.com/stateful/runme/v3/internal/client/graphql"
-	"github.com/stateful/runme/v3/internal/cmd"
-	"golang.org/x/oauth2"
 )
 
 var (
-	apiURL   = flag.String("api-url", "https://api.stateful.com", "The API base address")
-	tokenDir = flag.String("token-dir", cmd.GetUserConfigHome(), "The directory with tokens")
+	apiURL = flag.String("api-url", "http://localhost:4000", "The API base address")
+	// tokenDir = flag.String("token-dir", cmd.GetUserConfigHome(), "The directory with tokens")
```

Comment:
Do we need this commented code?

Reply:
As said in the description, some of this code is temporarily disabled but probably has to be brought back as part of actually having a client/consumer rely on it.